index.bs

```diff
@@ -1503,12 +1503,17 @@ exposing [=low-level=] sensor APIs, but should also expose
 
 <h3 id="multiple-sensors">When is Enabling Multiple Sensors of the Same Type Not the Right Choice?</h3>
 
-TODO: provide guidance on when to:
+It is possible to construct multiple {{Sensor}} instances of the same [=sensor type=], however,
+it is not advisable to do that when these instances are constructed using equal {{SensorOptions}},
```

Comment:
"it is not advisable to have multiple {{Sensor}} objects that were constructed with the same construction parameters as it can unnecessarily increase CPU/power load. Instead, multiple observers should add event listeners to a single {{Sensor}} object. " wdyt?